src/bundle/Resources/views/admin/content_draft/draft_list.html.twig

```diff
@@ -0,0 +1,179 @@
+{% extends "@ezdesign/layout.html.twig" %}
+
+{% trans_default_domain 'drafts' %}
+
+{% form_theme form_remove '@ezdesign/form_fields.html.twig' %}
+
+{% import _self as macros %}
+
+{% block title %}{{ 'drafts.list'|trans|desc('Drafts') }} - {{ parent() }}{% endblock %}
+
+{% block body_class %}ez-drafts-list-view{% endblock %}
+
+{% block content %}
+    <div class="row align-items-stretch ez-main-row">
+        {% block left_sidebar %}
+            {{ parent() }}
+        {% endblock left_sidebar %}
+
+        <div class="px-0 ez-content-container">
+            <section class="container mt-5">
+                {% include '@ezdesign/parts/page_title.html.twig' with {
+                    title: 'drafts.list'|trans|desc('Drafts'),
+                    iconName: 'content-draft'
+                } %}
+
+                <div class="px-4">
+                    {{ form_start(form_remove, {
+                        'action': path('ezplatform.content_draft.remove'),
+                        'attr': {
+                            'class': 'ez-toggle-btn-state',
+                            'data-toggle-button-id': '#confirm-' ~ form_remove.remove.vars.id
+                        }
+                    }) }}
+
+                    {% include '@ezdesign/parts/table_header.html.twig' with {
+                        ground: 'mt-3',
+                        headerText: 'drafts.table.header'|trans|desc('Drafts') ~ ' (' ~ pager.count ~ ')',
+                        tools: macros.table_header_tools(form_remove)
+                     } %}
+                    <div class="ez-scrollable-table-wrapper">
+                        <table class="ez-table table">
+                            <thead>
+                                <tr>
+                                    <th class="ez-table__header-cell"></th>
+                                    <th class="ez-table__header-cell ez-table__header-cell--has-icon"></th>
+                                    <th class="ez-table__header-cell ez-table__header-cell--after-icon">{{ 'drafts.list.name'|trans|desc('Name') }}</th>
+                                    <th class="ez-table__header-cell">{{ 'drafts.list.content_type'|trans|desc('Content Type') }}</th>
+                                    <th class="ez-table__header-cell">{{ 'drafts.list.modified_language'|trans|desc('Modified Language') }}</th>
+                                    <th class="ez-table__header-cell">{{ 'drafts.list.version'|trans|desc('Version') }}</th>
+                                    <th class="ez-table__header-cell">{{ 'drafts.list.last_saved'|trans|desc('Last Saved') }}</th>
+                                    <th class="ez-table__header-cell"></th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                            {% if pager.count is same as(0) %}
```

Comment:
This construction looks quite weird to me. Could you use `for-else` instead? Ref.: https://twig.symfony.com/doc/2.x/tags/for.html#the-else-clause